src/gui/transferlistdelegate.h

```diff
@@ -26,24 +26,23 @@
  * exception statement from your version.
  */
 
-#ifndef TRANSFERLISTDELEGATE_H
-#define TRANSFERLISTDELEGATE_H
+#pragma once
```

Comment:
Most of the changes in this file belong conceptually to the previous commit.
However, I am not insisting on changing this.

Reply:
Sorry, what do you mean exactly?
The second commit includes only changing from QItemDelegate to QStyledItemDelegate and some code/style cleanups (including this one).

Reply:
pragma/explicit/Q_DISABLE_COPY could go to the previous commit, like it is done for the sort model (IIRC).
But it is ok. I am bothered to much by this.